RTN-011.tex

```diff
@@ -47,9 +47,13 @@
 % Leanne 
 \setDocAbstract{%
 Science during the first year of Rubin Observatory / LSST operations is a high priority. 
-Alerts to  of transient, variable, and/or moving objects. are the only data product that will be immediately available (within 60 seconds of image readout) and publicly shareable, i.e not subject to a proprietary period. 
-The worldwide community is actively preparing to process the LSST alert stream and use it to generate groundbreaking scientific results. This note describes the \ro plan for ensuring \es. It is a living document that will evolve over the course of the remainder of the construction project and up until \drone. 
-
+Prior to \drone, scientific investigations may be undertaken with data processed in Alert Production and Solar System Processing as well as \dpvs of the commissioning data.
+Alerts of transient, variable, and/or moving objects are the only data product that will be immediately available (within 60 seconds of image readout) and publicly shareable, i.e not subject to a proprietary period. 
+Alert Production requires high-quality template images of the sky, and thus full volume and full fidelity alerts will not be available until after \drone.
+However, \ro is planning a process of incremental template generation to maximize the time-domain and solar system science achievable in the first year of operations.
+The worldwide community is actively preparing to process the LSST alert stream and use it to generate groundbreaking scientific results. 
+This note describes the \ro plan for ensuring \es. 
+It is a living document that will evolve over the course of the remainder of the construction project and up until \drone. 
 }
 
```

Comment:
Reads well. In another ticket, I've updated the abstract to be less detailed and added a summary section. I will work your version into this.